lib/accept_terms.rb

```diff
@@ -0,0 +1,27 @@
+class AcceptTerms
+  # Number of objects to create before creating a new transaction
+  OBJECTS_PER_TRANSACTION = 1000
+
+  def initialize(users)
+    @users = users
+  end
+
+  def run
+    i = 0
+    contracts = FinePrint::Contract.all
+    while true do
+      users = @users.limit(OBJECTS_PER_TRANSACTION)
```

Comment:
Instead of manually doing batch queries, just use `find_each` -- does the same thing.

http://api.rubyonrails.org/classes/ActiveRecord/Batches.html